doc/source/whatsnew/v0.24.2.rst

```diff
@@ -62,7 +62,7 @@ Bug Fixes
 
 **Categorical**
 
--
+- Constructing a `Categorical` from a `Series` could cause segfaults (:issue:`25318`)
```

Comment:
Can you move this to the regressions section above?

Reply:
be more specific about this it is too general a statement